app/alarm/ui/src/main/java/org/phoebus/applications/alarm/ui/table/AlarmTableUI.java

```diff
@@ -277,7 +282,12 @@ private ToolBar createToolbar()
         search.setTooltip(new Tooltip("Enter pattern ('vac', 'amp*trip')\nfor PV Name or Description,\npress RETURN to select"));
         search.textProperty().addListener(prop -> selectRows());
 
-        return new ToolBar(active_count,ToolbarHelper.createStrut(), ToolbarHelper.createSpring(), server_mode, acknowledge, unacknowledge, search);
+	if (AlarmSystem.disable_notify_visible.equals("true"))
```

Comment:
disable_notify_visible should be a boolean that can simply be compared as `if (...disable_notify_visible)`

Reply:
Done